core/amber/src/test/scala/edu/uci/ics/texera/workflow/operators/hashJoin/HashJoinOpExecSpec.scala

```diff
@@ -90,6 +92,8 @@ class HashJoinOpExecSpec extends AnyFlatSpec with BeforeAndAfter {
       build,
       "same",
       "same",
+      false,
+      false,
```

Comment:
can we add 3 more test cases? for left outer, right outer, and full outer? thanks

Reply:
added 1 simple test case for full outer join. we should add more concrete and more test cases in the future.